examples/server_to_splunk.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+"""
+Example of sending data to Splunk.
+Documentation related to the script is available in Cyberwatch: https://[CYBERWATCH_URL]/help/en/5_connect_cyberwatch_third_party/import_data_siem.md
+"""
+
+import requests
+import os
+from six.moves.configparser import ConfigParser
+from cbw_api_toolbox.cbw_api import CBWApi
+
+def post_splunk(url, token, payload):
+    """
+        Send a post request to Splunk API
```

Comment:
Describe the method before its declaration.

Reply:
https://www.python.org/dev/peps/pep-0257/

Reply:
My bad !